tools/mafft/mafft.xml

```diff
@@ -60,35 +80,73 @@
             <when value="mafft-linsi"/>
             <when value="mafft-qinsi"/>
             <when value="mafft-xinsi"/>
+            <when value="custom">
+                <param name="distance_method" type="select" display="radio" label="Distance method" help="Distance method must be chosen regarding your data">
+                    <option value="--6merpair" selected="true">Shared 6mers distance (fastest)</option>
+                    <option value="--globalpair">Global alignment (Needleman-Wunsch)</option>
+                    <option value="--localpair">Local alignment (Smith-Waterman)</option>
+                    <option value="--genafpair">Local, affine gap cost</option>
+                </param>
+                <param name="ep" type="float" value="0.123" label="Offset value" help="Offset value, which works like gap extension penalty, for group-to-group alignment" />
+                <param name="retree" type="integer" value="2" min="1" max="100" label="Guide tree is built this number of times in the progressive stage." help="Valid with 6mer distance" />
+                <param name="iterations" type="integer" value="0" min="0" max="1000" label="Maximum number of iterations" help="1000 for maximum quality" />
+            </when>
         </conditional>
+        <param name="op" type="float" value="1.53" label="Gap opening penalty" help="1.53 default value" />
```

Comment:
Is there a reason for "ep" being inside custom MAFFT flavour while "op" is here?